Penumbra/Import/Models/Export/MeshExporter.cs

```diff
@@ -427,23 +442,46 @@ private IVertexMaterial BuildVertexMaterial(IReadOnlyDictionary<MdlFile.VertexUs
 
         if (_materialType == typeof(VertexTexture2ColorFfxiv))
         {
-            var uv = ToVector4(attributes[MdlFile.VertexUsage.UV]);
+            var uv = ToVector4(attributes[MdlFile.VertexUsage.UV].First());
             return new VertexTexture2ColorFfxiv(
                 new Vector2(uv.X, uv.Y),
                 new Vector2(uv.Z, uv.W),
-                ToVector4(attributes[MdlFile.VertexUsage.Color])
+                ToVector4(attributes[MdlFile.VertexUsage.Color].First())
+            );
+        }
+        if (_materialType == typeof(VertexTexture3))
+        {
+            // Not 100% sure about this

```

Comment:
makes sense on first blush, though i suppose there's nothing forcing it to remain as such. is there an easy way to detect when it may be incorrect to fail out? ig inherent issue is that unused values could be valid-looking zeros or junk, so detection might be a crapshoot.